.github/workflows/easy-cp.yml

```diff
@@ -47,29 +47,26 @@ jobs:
           git config user.email "<>"
           git fetch origin $RELEASE_BRANCH
           git fetch origin master
-          git checkout -b $RELEASE_BRANCH --track origin/$RELEASE_BRANCH
+          git checkout -b cp-${CHANNEL}-${COMMIT_SHA} --track origin/$RELEASE_BRANCH
```

Comment:
If these are being created under a bot account, does origin need to be changed to upstream/$RELEASE_BRANCH?

Reply:
When we use the marketplace checkout package `actions/checkout`, the `origin` is set to flutter/flutter. 

If we get rid of dependency on market place checkout, and if we use a fork of the repository, then we can `git remote set origin` to be the fork, and `git remote set upstream` to be flutter/flutter. Currently we are not using a fork and are using market place action, `origin` would be flutter/flutter and `upstream` is unset.